mender-convert-modify

```diff
@@ -225,19 +225,34 @@ EOF
 log_info "Installing Mender client and related files"
 
 if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
+    MENDER_CLIENT_DEB_VERSION="$MENDER_CLIENT_VERSION"
```

Comment:
Then I suggest we rename this lowercase - we use uppercase for configuration or "global" variables.
```suggestion
    mender_client_deb_version="$MENDER_CLIENT_VERSION"
```